crates/solidity/src/lib.rs

```diff
@@ -46,6 +46,9 @@ pub use self::solc::version::Version as SolcVersion;
 pub use self::solc::Compiler;
 pub use self::version::Version as ResolcVersion;
 pub use self::warning::Warning;
+
+pub use solc::{FIRST_SUPPORTED_VERSION, LAST_SUPPORTED_VERSION};
```

Comment:
I know it is a bit arbitrary but at least we should try to keep some consistency to not make this a complete mess.

Please export those like the rest, meaning like this (and move them up, the formatter should then sort those automatically to the correct position):

```
pub use self::solc::FIRST_SUPPORTED_VERSION as SolcFirstSupportedVersion;
pub use self::solc::LAST_SUPPORTED_VERSION as SolcLastSupportedVersion;
```

Reply:
done